server/security.go

```diff
@@ -75,71 +72,30 @@ func (a *ApplicationHandler) CreateAPIKey(w http.ResponseWriter, r *http.Request
 // @Success 201 {object} util.ServerResponse{data=models.PortalAPIKeyResponse}
 // @Failure 400,401,500 {object} util.ServerResponse{data=Stub}
 // @Security ApiKeyAuth
-// @Router /ui/organisations/{orgID}/groups/{groupID}/apps/{appID}/keys [post]
+// @Router /ui/organisations/{orgID}/security/applications/{appID}/keys [post]
 
-// CreateAppAPIKey
-// @Summary Create an api key for app portal or the cli (API)
-// @Description This endpoint creates an api key that will be used by app portal or the cli
+// CreateAppPortalAPIKey
+// @Summary Create an api key for app portal (API)
+// @Description This endpoint creates an api key that will be used by app portal
 // @Tags APIKey
 // @Accept  json
 // @Produce  json
 // @Param appID path string true "application ID"
 // @Success 201 {object} util.ServerResponse{data=models.PortalAPIKeyResponse}
 // @Failure 400,401,500 {object} util.ServerResponse{data=Stub}
 // @Security ApiKeyAuth
-// @Router /security/applications/{appID}/keys [post]
-func (a *ApplicationHandler) CreateAppAPIKey(w http.ResponseWriter, r *http.Request) {
```

Comment:
Why did we take this guy out? @Youngestdev @danvixent

Reply:
Ngl, I don't remember cleaning this 😓